server/fleet/software.go

```diff
@@ -436,3 +436,10 @@ type VPPBatchPayloadWithPlatform struct {
 	SelfService bool                `json:"self_service"`
 	Platform    AppleDevicePlatform `json:"platform"`
 }
+
+type FleetMaintainedAppAvailable struct {
```

Comment:
is there another type for `FleetMaintainedApp`? wondering if we should drop the `Available` as this is just the public-facing version of a fleet maintained app

Reply:
I was going to comment on this; why not use the existing `MaintainedApp` that Martin added?

Reply:
It contains a a couple of fields not required by the endpoint, and I started working on this PR before his was merged so that type wasn't available. Won't be much work to use it instead